CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Changelog
 
+## [1.142.5] - 2024-08-14
+
+### CI
+
+- Configure Dependabot to update GitHub Actions.
+
+### Documentation
+
+- Document new `mdns_enabled` behavior.
+
+### Fixes
+
+- Still try to create "INBOX.DeltaChat" if couldn't create "DeltaChat" ([#5870](https://github.com/deltachat/deltachat-core-rust/pull/5870)).
+- Store_seen_flags_on_imap: Skip to next messages if couldn't select folder ([#5870](https://github.com/deltachat/deltachat-core-rust/pull/5870)).
```

Comment:
Where this capitalisation comes from? The function is called `store_seen_flags_on_imap` in the commit message